pkg/api/customization/authn/rtb_store.go

```diff
@@ -26,7 +26,9 @@ func SetRTBStore(ctx context.Context, schema *types.Schema, mgmt *config.ScaledC
 			if id, ok := data[client.ClusterRoleTemplateBindingFieldUserID].(string); ok && id != "" {
 				u, err := userLister.Get("", id)
 				if err != nil {
-					logrus.Errorf("problem retrieving user for CRTB %v from cache during CRTB transformation: %v", data, err)
+					if !errors.IsNotFound(err) {
```

Comment:
all other changes in this commit are effectively formatting or fixes to error shadowing. this is not. whats the reason/need for it? It should be mentioned in the commit message.

Reply:
I believe this change should be made because everytime we delete a user it dumps out the entire object. This would be useful for actual errors but is not needed when the root cause of the error is the user not being in the cache. 
Another possible change would be to change it to print with an info status that the user was not found instead of printing the large object